cmux-tui/bindings/python/cmux/raw/client.py

```
@@ -430,6 +430,16 @@ def _open_command_stream(self, command: str, request: Any) -> _Stream:
         stream_type = AttachStream if metadata.stream_kind == "attach" else EventStream
         return stream_type(self, command, payload)
 
+    def _open_command_stream_with_result(self, command: str, request: Any) -> tuple[Any, _Stream]:
```

Ruff (0.16.3)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `request`

(ANN401)